types/npmlog/npmlog-tests.ts

```diff
@@ -42,3 +44,20 @@ npmlog.addLevel("styled-level", 42, {
     bold: false,
     underline: true,
 }, 'display name');
+
+npmlog.addLevel("broadcast", 10);
+
+npmlog.broadcast(prefix, message);
+npmlog.broadcast(message);
+
+npmlog.on("broadcast", () => {});
+
+const msg: MessageObject = {
```

Comment:
Fix error by writing npmlog.MessageObject here instead of adding another import?

Reply:
Thanks - done!